draft-jasdips-regext-rdap-rpki.md

```diff
@@ -767,8 +760,10 @@ type is "rpki1_aspa".
 
 ## Relationship with Autonomous System Number Object Class
 
-It would be useful to show all the ASPAs associated with an autonomous system number object. To that end, this extension
-adds a new "rpki1_aspas" member to the Autonomous System Number object class ([@!RFC9083, section 5.5]):
+An autonomous system number object for an ASN range can span multiple ASPA objects. However, an ASPA object can only be
+linked to a single autonomous system number object. It would be useful to show all the ASPA objects associated with an
```

Comment:
Is it clear in this sentence that an ASPA object can only contain the providers (potentially many ASNs) of one specific ASN?

Both provider and customer feel "associated" to an ASN object, while we probably only intend to list the customer AS ASPAs (that are signed by this AS)

Reply:
>Is it clear in this sentence that an ASPA object can only contain the providers (potentially many ASNs) of one specific ASN?

No. Will try clarifying that the autonomous system number objects cover customer ASes vis-a-vis ASPAs.

> Both provider and customer feel "associated" to an ASN object, while we probably only intend to list the customer AS ASPAs (that are signed by this AS)

Right. The ASN registration holder is cryptographically asserting from the customer AS perspective, about its relationship with providers ASes.